<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OltPageWidget</class>
 <widget class="QWidget" name="OltPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="ontListTitleLabel">
         <property name="text">
          <string>Список [NTX]</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ontFilterHorizontalLayout">
         <item>
          <widget class="QComboBox" name="filterOntComboBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Mac/Serial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Описание</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterOntLineEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Фильтрация по MAC-адресу/Серийному номеру...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="ontListTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="userSettingsGroupBox">
         <property name="title">
          <string>ACS настройки пользователя [NTP-1 xxx] </string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="pppSettingsGroupBox">
            <property name="title">
             <string>Настройки PPP</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="pppLoginLabel">
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="pppLoginLineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="pppPasswordLabel">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="pppPasswordLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="sipSettingsGroupBox">
            <property name="title">
             <string>Настройки SIP</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="1">
              <widget class="QLineEdit" name="sipProxyLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="sipProxyLabel">
               <property name="text">
                <string>SIP прокси</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QGroupBox" name="sipPhone2GroupBox">
               <property name="title">
                <string>SIP Phone 2</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <layout class="QFormLayout" name="formLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="loginPhone2Label">
                  <property name="text">
                   <string>Login</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="passwordPhone2Label">
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="loginPhone2LineEdit"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="passwordPhone2LineEdit"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QGroupBox" name="sipPhone1GroupBox">
               <property name="title">
                <string>SIP Phone 1</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <item row="1" column="0">
                 <widget class="QLabel" name="loginPhone1Label">
                  <property name="text">
                   <string>Login</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="loginPhone1LineEdit"/>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="passwordPhone1LineEdit"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="passwordPhone1Label">
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="applyCredentialsButton">
            <property name="text">
             <string>Применить изменения</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="rightVerticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="macAddressTitleLabel">
         <property name="text">
          <string>Таблица MAC-адресов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filterGroupBox">
         <property name="title">
          <string>Фильтрация</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="macRadioButton">
            <property name="text">
             <string>по MAC-адресу</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="macLineEdit"/>
          </item>
          <item>
           <widget class="QRadioButton" name="portRadioButton">
            <property name="text">
             <string>по портам</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portListLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterPortButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Отфильтровать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="macAddressTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
